backend/main.py

```
@@ -1,30 +1,65 @@
+from typing import Annotated
 from contextlib import asynccontextmanager
 
-from fastapi import FastAPI
+from fastapi import Depends, FastAPI, HTTPException, Query
```

Ruff (0.11.9)

`fastapi.Query` imported but unused

Remove unused import: `fastapi.Query`

(F401)